hw/ip/rstmgr/rstmgr_cnsty_chk.core

```diff
@@ -0,0 +1,40 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: "lowrisc:ip:rstmgr_cnsty_chk"
+description: "Rstmgr consistency checker"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:prim:assert
+      - lowrisc:prim:sparse_fsm
+      - "fileset_ip     ? (lowrisc:ip:rstmgr_pkg)"
+      - "fileset_top    ? (lowrisc:systems:rstmgr_pkg)"
```

Comment:
This appears to be missing some deps for top-autogeneration (only needed for the englishbreakast).
Adding 
```
      - "fileset_top    ? (lowrisc:systems:rstmgr)"
      - "fileset_topgen ? (lowrisc:systems:topgen)"
 ```

should do the trick.

Reply:
That is funny: the added deps do not seem real dependencies, meaning the rstmgr_cnsty_chk module should not depend on rstmgr, or topgen... But trying it out anyway :)

Reply:
yeah you are right. I think you only need the `pkg` ones i'm..guessing.
Do you even know those? Because if there are no references, maybe we can just remove them.